src/operators/transformer.operators.ts

```diff
@@ -3,6 +3,22 @@ import { plainToClass, Exclude } from 'class-transformer';
 
 import { Obj } from './derive.operator';
 
+/* Return the available keys for a class.
```

Comment:
I factored out this "get keys for class" function so I could document it more clearly.